### PR TITLE
Homepage -> LoginForm Styling

### PR DESCRIPTION
**Pull request by emrerdem1**
Hey Louis! I worked on styling of loginForm a bit more, now it fits better with our general theme.

Kindly let me know if you see anything wrong.

### CODE REVIEW

**Comment by emrerdem1** [comment]
PTAL

File: src/components/auth/login/GoogleAndFacebookLogIn.jsx [hunk 1]
```diff
@@ -39,27 +37,16 @@ export default function GoogleAndFacebookLogIn() {
...
-          variant="info"
-          size="md"
-          className="googleButton"
-          onClick={() => logIn(googleProvider)}
-        >
-          Google
-        </Button>
-      </Col>
+    <Row className="socialLoginIcons">
+      <span>Login/Register using:</span>
```
Comment (louisrli) [inline src/components/auth/login/GoogleAndFacebookLogIn.jsx]: Login or register

Comment (louisrli) [reply src/components/auth/login/GoogleAndFacebookLogIn.jsx]: using:

File: src/components/layout/nav/Navbar.scss [hunk 2]
```diff
@@ -1,5 +1,9 @@
 @import "../../common/variables";
 
+$navbarIndex: 102;
```
Comment (louisrli) [inline src/components/layout/nav/Navbar.scss:4]: it's too late to fix this but I'll say in general if you're using z indexes something weird is going on, especially like this. that it to say I almost never have to use z indexes except in a few rare cases. be more careful of this in the future

Comment (louisrli) [reply src/components/layout/nav/Navbar.scss:4]: you should add some fairly descriptive comments above this explaining why this is needed. someone who knows nothing about your code should be able to understand by reading said comments

File: src/components/layout/nav/signup/SignUp.jsx [hunk 3]
```diff
@@ -9,11 +10,12 @@ const SignUp = () => {
   return (
     <>
       <Col xl={11} className="signupBox">
-        <GoogleAndFacebookLogIn />
         <Login />
         <Col className="signupRedirect">
-          <NavLink to="/signup">Still not a user? Sign up now!</NavLink>
+          <NavLink to="/signup">Still not a user? Register now!</NavLink>
         </Col>
+        <Row className="socialMediaSeperator"></Row>
```
Comment (louisrli) [inline src/components/layout/nav/signup/SignUp.jsx]: separator is spelled wrong

**Review by louisrli — CHANGES_REQUESTED** [review]

File: src/components/layout/nav/Navbar.scss (hunk 2 above)
Comment (emrerdem1) [reply src/components/layout/nav/Navbar.scss:4]: Hmm, anladım.

**Comment by emrerdem1** [comment]
PTAL

**Review by louisrli — APPROVED** [review]

File: src/components/layout/nav/signup/SignUp.jsx [hunk 4]
```diff
@@ -9,11 +10,12 @@ const SignUp = () => {
   return (
     <>
       <Col xl={11} className="signupBox">
-        <GoogleAndFacebookLogIn />
         <Login />
         <Col className="signupRedirect">
-          <NavLink to="/signup">Still not a user? Sign up now!</NavLink>
+          <NavLink to="/signup">Still not a user? Register now!</NavLink>
         </Col>
+        <Row className="socialMediaSeparator"></Row>
```
Comment (OmarAlithawi) [inline src/components/layout/nav/signup/SignUp.jsx:17]: Why did u use an empty Row in here , and the parent is  Col without any Container or Row ?

File: src/components/auth/login/GoogleAndFacebookLogIn.jsx [hunk 5]
```diff
@@ -39,27 +37,16 @@ export default function GoogleAndFacebookLogIn() {
...
-          size="md"
-          className="googleButton"
-          onClick={() => logIn(googleProvider)}
-        >
-          Google
-        </Button>
-      </Col>
+    <Row className="socialLoginIcons">
+      <span>Login or Register using:</span>
+      <i
```
Comment (OmarAlithawi) [inline src/components/auth/login/GoogleAndFacebookLogIn.jsx:41]: why did you use a Row without a Col ?

File: src/components/layout/nav/signup/SignUp.jsx (hunk 4 above)
Comment (emrerdem1) [reply src/components/layout/nav/signup/SignUp.jsx:17]: it's separator line with a text that is centered, you can check it on netlify deploy.

File: src/components/auth/login/GoogleAndFacebookLogIn.jsx (hunk 5 above)
Comment (emrerdem1) [reply src/components/auth/login/GoogleAndFacebookLogIn.jsx:41]: What do you mean? I didn't understand.

File: src/components/layout/nav/signup/SignUp.jsx (hunk 4 above)
Comment (emrerdem1) [reply src/components/layout/nav/signup/SignUp.jsx:17]: We don't have to use .container and .row wrapper just to use .col, right? Same goes here too, is it wrong?

File: src/components/auth/login/GoogleAndFacebookLogIn.jsx (hunk 5 above)
Comment (emrerdem1) [reply src/components/auth/login/GoogleAndFacebookLogIn.jsx:41]: Col could be used as well but didnt want to add too many unnecessary wrapper.

**Review by OmarAlithawi — APPROVED** [review]

**Comment by louisrli** [comment]
lgtm

**Comment by louisrli** [comment]
you need to make a new pr

**Comment by louisrli** [comment]
oh nvm maybe I commented the wrong one

**Pull request merged**